src/validations/admins.js

```diff
@@ -0,0 +1,36 @@
+const Joi = require('joi');
+
+const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/;
+const numberDNILength = (number, helpers) => {
+  if (number.toString().length === 9) {
+    return number;
+  } return helpers.message('Dni must be 9 numbers');
+};
+const numberPhoneLength = (number, helpers) => {
+  if (number.toString().length === 10) {
+    return number;
+  } return helpers.message('Phone must be 10 numbers');
+};
+
+const validateAdminUpdate = (req, res, next) => {
+  const adminValidation = Joi.object({
+    firstName: Joi.string().min(3).max(15),
+    lastName: Joi.string().min(3).max(15),
+    dni: Joi.number().custom((number, helpers) => (numberDNILength(number, helpers))),
```

Comment:
If we set a string data, we do not need to set a custom validation because that make with string length